tests/test_qa_c2_phase_settlement_mask_1374.py

```
@@ -130,7 +130,8 @@ def _submit_hitl(choices, *, write_gate, on_event=None, cheat_directive=""):
     async def _go():
         # 并行：放行 phase2 前先观测展示态
         async def _watch():
-            assert await asyncio.get_event_loop().run_in_executor(None, phase2_started.wait, 5.0)
+            # bare 事件屏障；禁 run_in_executor(wait, 墙钟)。
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)